### PR TITLE
mitosheet: add prompt that never is too long

### PR DESCRIPTION
**Pull request by naterush**
# Description

Adds a `get_prompt` function that always is within the length bounds, intelligently. We will, of course, continue to improve this over time.

# Testing

Add some other tests.

# Documentation

No.

### CODE REVIEW

File: mitosheet/mitosheet/ai/prompt.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+```"""
+    else:
+        return f"""Transformation: replace 1 with 2 in df
+Code:
+```
+df.replace(1, 2, inplace=True)
+```"""
+
+
+def get_prompt(df_names: List[str], dfs: List[pd.DataFrame], current_selection: Optional[Selection], user_input: str) -> str:
```
Comment (aarondr77) [inline mitosheet/mitosheet/ai/prompt.py:126]: This only supports a single selection, but we let users have many selections. Do we want to support that? ie: "Apply this formula to all selected columns" could be a common command?

**Review by aarondr77 — COMMENTED** [review]
Nice! This was pleasant to review since its such a small chunk!

File: mitosheet/mitosheet/ai/prompt.py (hunk 1 above)
Comment (naterush) [reply mitosheet/mitosheet/ai/prompt.py:126]: The selection object on the backend is very different than frontend. We're just going to send all the selected columns and rows to the backend, and that will be considered one selection.

**Comment by naterush** [comment]
This is ready to merge when tests pass.

**Pull request merged**